source/develop/release-management/features/storage/pass_discard_from_guest_to_underlying_storage.html.md

```diff
@@ -0,0 +1,234 @@
+---
+authors: ishaby
+wiki_title: Features/PassDiscardFromGuestToUnderlyingStorage
+feature_name: Pass discard from guest to underlying storage
+feature_modules: engine/vdsm/api-model
+feature_status: Development
+---
+
+# Pass discard from guest to underlying storage
+
+## Summary
+
+### How we use thin provisioned disks today
+Today, when we create a thin provisioned disk on a block domain, a logical volume with a size of 1G is created, regardless its virtual size.<br/>
+When the free space is about to end, we extend the lv with one more Gigabyte, so that the disk grows again and again until it reaches its virtual size.<br/>
+If the lun that is used to create the storage domain is also thin provisioned, it behaves in a similar way. That is, when we extend our disk with one Gigabyte, we don't necessarily get 1G from the storage, but we do get 1G of virtual space.
```

Comment:
I don't understand this sentence. The mechanisms are quite different. Can you elaborate?